cloud_edition/flexibility_mode/docs/composer_settings.rst

```diff
@@ -0,0 +1,58 @@
+Composer settings
+=================
+
+
+
+| During the install process we used our own repository for the EE distribution. But you don't have any access to this repository after the install process.
+| We setup your own distribution repository in the composer file, so you can change it if you want.
+
+
+| To be able to pull from distribution, you must use a known ssh keypair from partner portal.
+| You can check your access rights to the distribution repository with this command:
+
+.. code-block:: bash
+
+    $ ssh git@distribution.akeneo.com -p 443
+    akeneo@my-instance-staging:~$ ssh git@distribution.akeneo.com -p 443
+    PTY allocation request failed on channel 0
+    hello papo-full_name, this is git@distribution running gitolite3 3.6-1~bpo70+1 (Debian) on git 1.7.10.4
+
+    R  	ElasticSearchBundle
+    R 	    InnerVariationBundle
+    R 	    LdapAuthenticationBundle
+    R 	    pim-enterprise-dev-partner
+
+
+| After that, you can set up your composer.json like that
+
+.. code-block:: json
+
+           "type": "vcs",
+           "url": "ssh://git@distribution.akeneo.com:443/pim-enterprise-dev-<who_i_am>.git",
+           "branch": "master"
+       }
+
+| All details about Distribution System can be found in the documentation “Akeneo EE Distribution System - User guide” in `the Partner portal`_.
+
+
+Composer and github api rate limit
+----------------------------------
+
+| The Community edition have many branches/tags and during a composer update process, you can reach easily the github api rate limit. Without OAuth token, github use the ip to manage api rate limit, and to limit to 60 requests per hour (https://developer.github.com/v3/#rate-limiting).
+
+| In order to update your pim, you must setup a github token for composer, you can run composer follow the instructions to help you.
```

Comment:
Incomprehensible sentance `you can run composer follow the instructions to help you` ??
Did you mean
```
| In order to update your pim, you must setup a github token for composer.
| Then you can run composer update and follow the instructions.
```